source/documentation/at-rules/import.html.md.erb

```diff
@@ -531,6 +531,56 @@ nice API for users of the new module system.
   @forward "reset" as reset-*
 <% end %>
 
+#### Configuring Modules Through Imports
+
+<% impl_status dart: '1.24.0', libsass: false, ruby: false %>
+
+You can configure modules that are loaded through an `@import` simply by
+defining global variables prior the `@import` that first loads that module.
+
+Note that changes to the configuration after the module is first loaded will be
+ignored.
+
+<% example do %>
+  // _library.scss
+  $color: blue !default;
+
+  a {
+    color: $color;
+  }
+  ---
+  // _library.import.scss
+  @forward 'library' as lib-*;
+  ---
+  // style.sass
+  $lib-color: green;
+  @import "library";
+  $lib-color: red;
+  @import "library";
```

Comment:
I think this is just going to produce more net confusion. Examples on the doc site don't have to be 100% comprehensive (that's what the spec is for), so it's more important that they show the most focused version of a feature possible.